packages/horizon/hardhat.config.ts

```diff
@@ -1,15 +1,30 @@
 import { hardhatBaseConfig } from 'hardhat-graph-protocol/sdk'
+import { HardhatUserConfig } from 'hardhat/config'
 
 // Hardhat plugins
 import '@nomicfoundation/hardhat-foundry'
 import '@nomicfoundation/hardhat-toolbox'
 import '@nomicfoundation/hardhat-ignition-ethers'
+import "@tenderly/hardhat-tenderly"
 import 'hardhat-storage-layout'
 import 'hardhat-contract-sizer'
-import 'hardhat-secure-accounts'
 
 if (process.env.BUILD_RUN !== 'true') {
   require('hardhat-graph-protocol')
 }
 
-export default hardhatBaseConfig
+const config: HardhatUserConfig = {
+  ...hardhatBaseConfig,
+  graph: {
+    deployments: {
+      ...hardhatBaseConfig.graph?.deployments,
+      subgraphService: false
```

Comment:
I think the cleanest approach is to only have horizon deployment in the base config, then each project would add whatever is of interest to them. Horizon is the common denominator for all data services after all.